client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -1,59 +1,47 @@
 import React, { Component } from 'react';
-// import Konva from 'konva';
-// import { Stage, Layer, Text, Circle, Line, Rect } from 'react-konva';
+import PropTypes from 'prop-types';
 
-import { initGridState, getFoods, getGhosts, getPacman, getWalls, getEnergizers, entityToCode } from './gameCore';
+import {
+  initGridState, getFoods, getGhosts,
+  getPacman, getWalls, getEnergizers,
+  entityToCode, entityApplier,
+} from './gameCore';
 
-import { PacmanBoard } from './PacmanBoard';
+import PacmanBoard from './PacmanBoard';
 
 class PacmanGame extends Component {
   constructor(props) {
```

Comment:
Use class variables instead of constructor

Reply:
Refactored with class variables